### ISSUE TITLE
Cron job

### ISSUE THREAD

**Issue by afromanex** [meffie/canton-tabletop.org#10]
We had talked briefly about a cron job to assist with creating content. Is there a cron job running somewhere already, or something that would need to be setup?

**Comment by meffie** [comment]
It would need to be setup. I was thinking we could do something to create weekly content for the game nights.

One possibility I just thought of would be to stage content as draft and then use a cron script to selectively change the draft true to false and then run hugo.

I've been using ollama (AI) to help with the text content, but it normally needs a bit of tweaking. Also, we are going to need a set of good board gaming images for the posts.

**Comment by afromanex** [comment]
Yes, maybe I could try and create some kind of utility/function/script to generate page content and then if that seems to come back with content reliably, we could maybe add it to a cron job?  

I'm thinking that with the need for images and human review of the content that maybe starting with something simple could at least be useful.
